File: Project Java & React/FrontEnd/src/Components/Services/cartservice.js

import httpCommon from "../../http-common"

class cartservice{

    addToCart(userId , item){
        return httpCommon.post('/cart/addtocart/'+userId ,item)
    }

    viewcart(userId){
        return httpCommon.get('/cart/getcart/'+userId)
    }

}

export default new cartservice

File: Project Java & React/BackEnd/demo/src/main/java/com/app/services/CartService.java
package com.app.services;

import com.app.dto.CartDto;
import com.app.dto.ItemRequest;

public interface CartService {
	
	public CartDto addToCart(ItemRequest item , int userId); 
	
	public CartDto getallcart(int userId);
	
	public CartDto removecartitems(String email, int foodid);
}
